prisma/schema.prisma

```diff
@@ -73,6 +73,11 @@ model Ballot {
     @@unique([voterId, roundId])
 }
 
+enum RoundType {
+    impact
+    project
+}
+
```

Comment:
The deployed schema already had this round type but there was a difference in the checked in schema.

Reply:
:wink: